### DISCUSSION TITLE
Version 25967 fixed taskbar and small icons setings, but not the icons.

### DISCUSSION THREAD

**Discussion by aldeniangman** [valinet/ExplorerPatcher#2372] [General]
IF YOU CLOSE THIS DISCUSSION ONE MORE TIME! I WILL BE SO ANGRY😡😡😡😡😡😡😡😡😡😡😡
_________________________________________________________________________________________________________________
so the icons that have wifi, battery, and other options are not showing.

**Comment by pyrates999** [comment]
That's a canary channel build https://blogs.windows.com/windows-insider/2023/10/05/announcing-windows-11-insider-preview-build-25967-canary-channel/

Unless you are a developer, you don't need to test beta versions of windows, you should only be running the release version of windows.

**Comment by Amrsatrio** [comment]
I am WELL aware. Do you want this broken clock in your taskbar?? So just, please, downgrade into Dev, Beta, or better release build like the normal people. I'm saying this for good. **What are you doing in Canary?** And why did you expect it to be flawless with EP? Although I've made EP not crash anymore and patch stuff properly, you must know that Microsoft has REMOVED the Windows 10 taskbar tray and Windows 10 start menu out of those canary builds. Adding them back without modifying system files can take a long time.

![image](https://github.com/valinet/ExplorerPatcher/assets/20662640/a884a928-5f2e-4ec1-a59c-906b86c4b40a)

Let's just continue discussing about this here. Sorry if I made you angry. https://github.com/valinet/ExplorerPatcher/discussions/2363
